src/library_manager/lib_manager.c

```diff
@@ -423,15 +423,13 @@ static int lib_manager_dma_buffer_alloc(struct lib_manager_dma_ext *dma_ext,
 	 * allocate new buffer: this is the actual DMA buffer but we
 	 * traditionally allocate a cached address for it
 	 */
-	dma_ext->dma_addr = (uintptr_t)rballoc_align(0, SOF_MEM_CAPS_DMA, size,
```

Comment:
@lyakh is this patch neededd for mtl-06 i.e. is it fixing a bug

Reply:
@lgirdwood I haven't seen any run-time failures that this patch would fix. I think "strangely" DMA on ADSP can work both with cached and uncached pointers, although it actually never uses cache. So I just find using cached pointers for DMA incorrect and confusing - it gives an impression of DMA using cache. So I think this is a correctness patch, but I don't know if it actually fixes anything. Besides this is lib-manager which isn't used yet anyway.